### PR TITLE
api to add users and score

### PR DESCRIPTION
**Pull request by Jeddaa**
Used the leader board API to make API calls to:

- generate a game Id
- add users and their score
- get all users and their scores

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,55 @@
...
+//         'Content-type': 'application/json; charset=UTF-8',
+//       },
+//     });
+//     const newData = await response.json();
+//     console.log(newData);
+//     return newData;
+//   } catch (e) {
+//     return e;
+//   }
+// }
```
Comment (DammyShittu) [inline src/index.js]: - [ ] Kindly remove this commented code.

File: src/modules/getGameId.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+async function getGameId() {
+  const gameId = 'V0cpkV3hkx9r4XWa2c0x';
+  localStorage.setItem('gameId', gameId);
+}
+export default getGameId;
```
Comment (DammyShittu) [inline src/modules/getGameId.js]: - [ ] Kindly use arrow functions for your method.

File: src/modules/refreshScore.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      newData.result.forEach((score) => {
+        const listItem = document.createElement('li');
+        listItem.classList.add('list-item');
+        listItem.textContent = `${score.user}: ${score.score}`;
+        list.appendChild(listItem);
+      });
+    } catch (e) {
+      console.error('Error adding new score:', e);
+    }
+  });
```
Comment (DammyShittu) [inline src/modules/refreshScore.js]: - [ ] After adding scores and clicking the refresh button the newly added scores are not added to the page. Kindly look into that and fix the issue.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Jeddaa,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/refreshScore.js (hunk 3 above)
Comment (thecodechaser) [inline src/modules/refreshScore.js]: - [ ] As requested by the previous reviewer, After adding a score and clicking on the refresh button the newly added score is not displaying on the screen. This is issue still not fixed. Please make sure to fix this issue.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Jeddaa👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Highly organized code. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/refreshScore.js (hunk 3 above)
Comment (Jeddaa) [reply src/modules/refreshScore.js]: hello @thecodechaser i don't really know the problem as the new score and user  is displaying on my end.

**Review by joel-kalema — COMMENTED** [review]
Hi @Jeddaa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**